Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -2486,8 +2519,14 @@ def fetch_incidents(client, first_fetch_time, integration_instance, last_run: di
     if incidents_from_previous_run:
         raw_incidents = incidents_from_previous_run
     else:
-        raw_incidents = client.get_incidents(gte_creation_time_milliseconds=last_fetch,
-                                             limit=max_fetch, sort_by_creation_time='asc')
+        if len(statuses) > 0:
```

Comment:
you should check for `statuses` for not being None. Otherwise this line will fail

Reply:
By default, `statuses` is an empty list.

Reply:
still, good practice to change this to `if statuses:` to cover all bases.